config.toml

```diff
@@ -1,5 +1,5 @@
 # Site settings
-baseurl = "https://uqcs.org/"
+baseURL = "https://uqcs.org/"
```

Comment:
Kind of a shame that Hugo breaks from the standard for not capitalising acronyms longer than two chars.

`baseURL` is correct in the Hugo sphere

Reply:
I think by shame here Jack means "breath of fresh air"

Reply:
`baseUrl` is superior 👀